TheJumpCloud/support
# Sa 2639 new jc import template dynamic populate template

## Issues
* [SA-2639](https://jumpcloud.atlassian.net/browse/SA-2639) - Force Param added to New-JCImportTemplate

## What does this solve?

This release adds a way for admins to quickly create new Import/ Update templates w/o going through the GUI options which were required in previous versions of the module. 

## Is there anything particularly tricky?

## How should this be tested?

**To test the New-JCImportTemplate Changes:**

Run:
```pwsh
# This will create a blank template to import new users
New-JCImportTemplate -Force -Type "Import"
# This will OVERWRITE the existing file and create a new update template containing all users in your org
New-JCImportTemplate -Force -Type "Update"
```

Next Run:
```pwsh
New-JCImportTemplate
```

The function should prompt users as normal and product a customized import/ update template depending on the selections.

**To test the Radius Server Changes:**
Create a new radius server and specify the new authIdp Parameter. It's an enum so tab keys should prompt you to enter one of the values "JUMPCLOUD" or "AZURE"
`New-JCRadiusServer -authIdp "AZURE"`

Next, try to change the name of the radius server you just created:
`Set-JCRadiusServer -id "YourRadiusServerID" -newName "NewRadiusServerName"`

The name should be updated and the authIdp type of "AZURE" should be retained.

Tests and issues with multiple Radius Servers returned `byName` to be resolved in SA-1375

## Screenshots
